include/veriblock/checks.hpp

```diff
@@ -23,6 +24,16 @@ namespace altintegration {
  */
 void checkRange(int64_t num, int64_t min, int64_t max);
 
+/**
+ * Checks if expression 'min' <= 'num' <= 'max' is true. If false, returns invalid state with error description.
+ * @param num number to check
+ * @param min min value
+ * @param max max value
+ * @param state will return error description here
+ * @return true if check is OK, false otherwise
+ */
+bool checkRangeNoExcept(int64_t num, int64_t min, int64_t max, ValidationState& state);
```

Comment:
```suggestion
bool checkRange(int64_t num, int64_t min, int64_t max, ValidationState& state);
```
I think there's no reason to add this to name